SplitFiles/control.c

```diff
@@ -165,7 +162,7 @@ void execute_command(ControlBuffer *ctrlBuff, GameParameters* params) {
         //Push some bread to a slot
         d = buffer_pop(ctrlBuff);
         e = buffer_pop(ctrlBuff);
-        if (d >= '0' && d <= '9' && (e == 'W' || e == 'B' || e == 'G'))  {
+        if (d >= '0' && d <= '9' && (e == 'W' || e == 'B' || e == 'G' || e == 'C'))  {
```

Comment:
Getting Ciabatta to toast needed code changing in at least three places - in a data-driven approach we want to define it once and have it all just work